### ISSUE TITLE
Webview cursor not changing type (or changing unreliably) since Qt 6.4.0 update

### ISSUE THREAD

**Issue by kleinerpirat** [ankitects/anki#2168]
Since commit 276c42e25 the cursor doesn't adjust to the types defined in our CSS, or does so very unreliably.

To test this, open the main view and move the mouse from the bottom up to the toolbar (e.g. to the Add button). It should swap to `pointer`, but stays default:

![image](https://user-images.githubusercontent.com/62722460/199363117-687e2e13-9949-433a-a44d-e58c895221d7.png)

If, however, you move from the top down into the toolbar, the cursor will change correctly:

![image](https://user-images.githubusercontent.com/62722460/199363136-a9163eed-ac2a-4a47-ab9c-bd6f8e823800.png)

This looks to me like Qt messed up a bit.

# Other issues
#### Editor split:
![image](https://user-images.githubusercontent.com/62722460/199363043-493f30db-0f76-4934-92d9-cd2a204f9a75.png)
### Field cursor:
![image](https://user-images.githubusercontent.com/62722460/199363228-a93c3031-5d46-4aec-9c2d-bf4e672640b8.png)

## What I tested
I removed my `cursor: pointer` override for Qt Widgets, but the issue persists regardless. There's no indication for a CSS issue either.

## How to fix this?
Honestly, I have no idea. The inner workings of Qt aren't something I'm familiar with.
I guess this is exclusive to Linux again? **Edit:** I tested the latest beta on Windows and found the same issues.

**Comment by glutanimate** [comment]
Yep, can confirm this also happens on macOS. Recorded these the other day (thus the old styling):


https://user-images.githubusercontent.com/5459332/199434052-dfa30b88-5a3f-4cbe-8d36-b978c1a5d6db.mov


https://user-images.githubusercontent.com/5459332/199434099-592d3fa3-d48e-4bac-8580-7bf01b902e2d.mov

Definitely seems like a weird Qt issue, perhaps something focus related?

**Comment by dae** [comment]
Qt 6.4.1 is expected in about 2 weeks; if the issue exists after it releases we should probably reproduce it with the Qt binary installer's sample apps and report the issue to them (though as it's relatively minor, not sure how much luck we'll have with them fixing it promptly)

**Comment by kleinerpirat** [comment]
We're on Qt 6.4.2 now and the issue hasn't been resolved yet. I'd say its worth reporting - it doesn't seem that minor to me.

---
Forum reports on this:
- https://forums.ankiweb.net/t/2-1-56-mouse-cursor-on-toolbar-is-inconsistent/26508
- https://forums.ankiweb.net/t/mouse-display-issues-in-2-1-57/26902

**Comment by dae** [comment]
Would you like to report this? Steps required are:

- Downloading Qt using https://download.qt.io/official_releases/online_installers/
- Opening Qt Creator in the install location
- Picking a sample app that is close to what is required to reproduce the issue. The markdown editor is probably useful here, as it allows you to enter arbitrary HTML
- Reproduce the problem outside of Anki, then file a report describing the steps required to reproduce the issue

An example: https://bugreports.qt.io/browse/QTBUG-76230

If you'd like me to report it instead, it would be great if you could provide a sample minimum HTML page that I can use.

**Comment by GithubAnon0000** [comment]
This has been solved recently. I think after the launcher has been introduced, one of the qt updates fixed the cursor behavior.

**Comment by StudentAnk** [comment]
Yes, I can confirm it's fixed. Not sure which specific update, but agree around the time of the new launcher. Happy :)
